public/stylesheets/site/2.0/26-media-narrow.css

```diff
@@ -122,6 +123,12 @@ body .narrow-shown {
   display: block;
 }
 
+/* 11 group: listbox */
```

Comment:
Without this, making the margins bigger would cut off part of the blurbs in the work and bookmark listboxes.